Apps/phdigitalshadows/digital_shadows.json

```diff
@@ -31,69 +33,77 @@
             "order": 0,
             "required": true
         },
-        "ph": {
-            "data_type": "ph",
-            "order": 5
-        },
         "ds_api_secret_key": {
             "data_type": "password",
             "description": "Digital Shadows API Secret Key",
             "order": 1,
             "required": true
         },
-        "history_days_interval": {
-            "data_type": "string",
-            "description": "History Data Interval (in days)",
-            "order": 4,
-            "required": true
-        },
         "private_incident": {
+            "data_type": "boolean",
+            "description": "Private Incidents for Ingest",
             "default": false,
             "order": 2,
-            "data_type": "boolean",
-            "description": "Private Incidents for Ingest"
+            "required":true
         },
         "global_incident": {
+            "data_type": "boolean",
+            "description": "Global Incidents for Ingest (Subscribed intelligence incidents will be pushed through private incident queue)",
             "default": false,
             "order": 3,
-            "data_type": "boolean",
-            "description": "Global Incidents for Ingest (Subscribed intelligence incidents will be pushed through private incident queue)"
+            "required":true
+        },
+        "history_days_interval": {
+            "data_type": "string",
+            "description": "History Data Interval (in days)",
```

Comment:
Description could be better. Should probably include example of value.

Reply:
Fixed